runtests.py

```diff
@@ -32,7 +32,7 @@ def main(run_slow=False, extra_args=None):
     """
     Run pytest tests for Spyder.
     """
-    pytest_args = ['spyder',
+    pytest_args = ['spyder/app/tests/test_mainwindow.py::test_debug_unsaved_file',
```

Comment:
Why is this change needed @impact27 ?

Reply:
I removed this commit. The test was failing and I was trying to understand what was happening. The tests should be passing now.

Reply:
👍